app/assets/javascripts/discourse/tests/acceptance/search-full-test.js

```diff
@@ -95,7 +96,12 @@ acceptance("Search - Full Page", function (needs) {
 
     server.put("/topics/bulk", (request) => {
       lastBody = helper.parsePostData(request.requestBody);
-      return helper.response({ topic_ids: [7] });
+      return helper.response({ topic_ids: [130] });
+    });
+
+    server.post("/search/click", () => {
+      searchResultClickTracked = true;
```

Comment:
We need to reset this after each test, otherwise we may be getting false positives.